tesseract-clickhouse/src/sql/aggregator.rs

```diff
@@ -61,6 +61,12 @@ pub fn agg_sql_string_pass_1(col: &str, aggregator: &Aggregator, mea_idx: usize)
                 join(secondaries, ", "),
             )
         },
+        Aggregator::Moe { .. }=> {
+            format!("sum(power( {} / 1.645, 2)) as m{}_moe_sub_agg",
```

Comment:
Maybe I am missing something, but should this `1.645` be substituted by the critical value?

Reply:
ah, yes, this one should be. thanks!